src/NotificationReceivedEvent.ts

```diff
@@ -3,13 +3,13 @@ import OSNotification from './OSNotification';
 const RNOneSignal = NativeModules.OneSignal;
```

Comment:
imported `Platform` is unused

Reply:
add `"noUnusedLocals": true` to tsconfig to catch these

Reply:
Done